Framework/TestHelpers/src/MuonWorkspaceCreationHelper.cpp

```diff
@@ -91,6 +96,12 @@ MatrixWorkspace_sptr createCountsWorkspace(size_t nspec, size_t maxt,
   return ws;
 }
 
+MatrixWorkspace_sptr createCountsWorkspace(size_t nspec, size_t maxt,
+                                           double seed, size_t detectorIDseed) {
+  return createCountsWorkspace(nspec, maxt, seed, detectorIDseed, true, 0.0,
```

Comment:
is this just calling itself?

Reply:
I think createCountsWorkspace here is overloaded so a different version is called depending on which inputs are used. This is the old interface which only took four inputs which then calls the new interface which takes six inputs. This also I think solves the default value issue above as the old tests will call this function which will then call the updated function with hist=true, xstart=0.0 and xend=1.0

Reply:
that makes sense.